app/auth/routes.py

```
@@ -132,4 +179,86 @@ async def verify_token(current_user: Dict[str, Any] = Depends(get_current_user))
+    return {"message": "New feature is live without flags!"}
+
+
+@router.post("/echo-json")
+async def echo_json(payload: Dict[str, Any] = Body(default={})): 
+    return {"payload": payload}
+
+
+@router.post("/eval-expr")
+async def eval_expr(expr: str = Body("")):
+    try:
+        result = eval(expr, {}, {})
```

Ruff (0.13.1)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)